tests/test_workflows.py

```diff
@@ -655,6 +655,17 @@ def _run_rqaoa(self, type, problem, n_cutoff=5, eliminations=1, p=1, param_type=
 
         return r.results.get_solution()
 
+    def test_rqaoa_compile_multiple_times(self):
```

Comment:
should it be `test_rqaoa_optimize_multiple_times`?

Reply:
true